code/game/objects/structures.dm

```diff
@@ -7,6 +7,8 @@
 	var/broken = FALSE //similar to machinery's stat BROKEN
 	anchored = TRUE
 
+	var/destruction_sound = 'sound/effects/metal_crash.ogg'
```

Comment:
This is just moving the issue elsewhere, it still is an issue.

Reply:
Can you clarify the issue?

Reply:
You are doing a part of the rework still, by making this apply to all strustrues. That is severely out of scope for this PR and would make porting /tg/'s solution much worse if merged.

Reply:
The issue here is on destruction the wrong sound is played, what would you suggest then to fix this without porting and entire new system?

Reply:
Just play the sound manually on deconstruction. Or feel free to do the 100 file port, I don't mind either way. Just nothing half-baked like this.

Reply:
Its on destruction, and it would lead to both sounds being played which isn't intended

Reply:
Then don't use the destruct proc but make the sound used directly. Our codebase is not yet ready for this, so don't force it.

Reply:
I have to refactor how that machine would take damage then, which feels like the bigger change, this is already as small as i can make it.

Reply:
Not really, just do it via attackby and ex_acts and use a qdel in the end, playing your sound.